pkg/cli/create_platform.go

```diff
@@ -164,7 +164,7 @@ func CreatePlatform(ctx context.Context, options *CreateOptions, globalFlags *fl
 	// check if we should connect to the vcluster or print the kubeconfig
 	if options.Connect || options.Print {
 		return ConnectPlatform(ctx, &ConnectOptions{
-			UpdateCurrent:         options.UpdateCurrent,
+			UpdateCurrent:         true,
```

Comment:
I think ConnectPlatform and UpdateKubeconfig above duplicate some of the connection functionality

Reply:
The `ConnectPlatform()` function ends with a `writkubeconfig()` function

Reply:
Good point. I checked the code and it looks like `ConnectPlatform` does call `platform.CreateVirtualClusterInstanceOptions` internally, so essentially doing the same as the block L148-L162 you mentioned above. I went ahead and remove this block and checked `create` in the different combinations (`--connect=true`, `--connect=false`, `--connect=false --print=true` once without a platform and once with being logged into the platform). Looks good so far.